src/components/PaymentLog.vue

```diff
@@ -698,6 +703,16 @@ export default {
   },
   data() {
     return {
+      dummyDiscount: {
```

Comment:
Do we want to leave this here?

Reply:
so that it can easily be changed if 7% is not appropriate

Reply:
is it coming from backend? the discount